File: backend/src/main/java/com/kh/backend/model/vo/Like.java
package com.kh.backend.model.vo;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class Like {

	private int likeNo;
	private int userNo;
	private char typeCode; // B, P
	private int refNo; // 게시글, 상품
}

File: service-frontend/src/type/like.ts
export interface Like{
    likeNo : number,
    userNo : number,
    typeCode : string,
    refNo : number
}

export const initLike:Like = {
    likeNo : 0,
    userNo : 10,
    typeCode : '',
    refNo : 0
}

export const initLikeList:Like[] = [];